### PR TITLE
Create sampleresources.md

### PR DESCRIPTION
**Pull request by garora**
Adding helpful sample.resources. So, users can get all links at one place.

### CODE REVIEW

File: sampleresources.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#Helpful sample resorces for Microsoft ASP.NET WebHooks
```
Comment (Eilon) [inline sampleresources.md]: Typo: `resorces` -> `resources`

File: sampleresources.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Please see the initial [documentation](http://docs.asp.net/projects/aspnetwebhooks/en/latest/) for details.
+
+Following(s) are helpful resources from our contributors:
```
Comment (Eilon) [inline sampleresources.md]: Can remove this whole line.

File: sampleresources.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#Helpful sample resorces for Microsoft ASP.NET WebHooks
+Please see the initial [documentation](http://docs.asp.net/projects/aspnetwebhooks/en/latest/) for details.
+
+Following(s) are helpful resources from our contributors: 
+
+##Samples
+- [GitHub](https://github.com/aspnet/WebHooks/tree/master/samples/#)
```
Comment (Eilon) [inline sampleresources.md]: Both of these sample URLs are 404's.

Comment (garora) [inline sampleresources.md]: These URLs will be automatically fixed, once current dev branch merged down to master.

Comment (Eilon) [inline sampleresources.md]: Ah, I see. What is the `GitHub` link then? Maybe just call it "All samples"? (I interpreted it to mean "A sample for using GitHub web hooks".)

File: sampleresources.md (hunk 1 above)
Comment (garora) [inline sampleresources.md]: Corrected!

File: sampleresources.md (hunk 2 above)
Comment (garora) [inline sampleresources.md]: removed in latest PR

**Comment by Eilon** [comment]
I also wonder if this is probably best to put in the main `Readme.md` - that already has a blog post link and it's probably the first place for people to look anyway. Having a separate file I think would reduce the chances of someone easily finding it.

**Comment by garora** [comment]
@Eilon - Initially, I was thinking the same. Then I thought to push this new file for review. I am updating these changes in ```Readme.md``` and afterward will delete ```sampleresources.md``` file. I appreciate for support & your time.

**Comment by garora** [comment]
@Eilon - Updated ```readme.md``` and removed ```sampleresources.md```.
All links are working and it looks good to me, request you to please review and do merge it down to your dev/master branch, if you find it fit.

**Comment by Eilon** [comment]
Was there a bad merge here? The PR looks messed up now, with a bunch of unrelated commits on it...

**Comment by garora** [comment]
Not sure, I just updated readme.md and deleted samplresources.md file.

**Comment by garora** [comment]
@Eilon- on the otherhand, if you suggest I will make new PR for Updates of. '''Readme.md'''.

**Comment by Eilon** [comment]
@garora that might be a good idea. Not sure what happened in this PR.

**Comment by garora** [comment]
Sure - I will do it. Thanks for your support and time

On Mon, Sep 28, 2015 at 12:10 PM, Eilon Lipton <notifications@github.com>
wrote:

> @garora <https://github.com/garora> that might be a good idea. Not sure
> what happened in this PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aspnet/WebHooks/pull/6#issuecomment-143653089>.
>



-- 
Thanks & Regards,
Gaurav Arora

**Comment by garora** [comment]
Closed PR. Created new [PR](https://github.com/aspnet/WebHooks/pull/7) .

**Pull request closed without merging**